treelib/tree.py

```diff
@@ -29,6 +29,11 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+try:
+    from builtins import str as text
+except ImportError:
+    from __builtin__ import str as text
```

Comment:
following this guideline: https://python-future.org/compatible_idioms.html#unicode